### ISSUE TITLE
Content: how to disable a user

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/guide#323]
It’s a simple one – we mention inactive users on https://guide.wagtail.org/en-latest/how-to-guides/manage-users-and-roles/. In the Concepts section, we have further details: https://guide.wagtail.org/en-latest/concepts/users-status/

I think we should:

1. Add a quick description of how to disable a user account by making it inactive, directly on https://guide.wagtail.org/en-latest/how-to-guides/manage-users-and-roles/.
2. Link to our Concepts page for more details.

**Comment by thibaudcolas** [comment]
@nouman17610 would you be ok to work on this one?

**Comment by nouman17610** [comment]
@thibaudcolas Yes please assign this issue to me. So I can start work on it. Thank you.

**Comment by nouman17610** [comment]
@thibaudcolas thank you

**Comment by nouman17610** [comment]
@thibaudcolas just wanted to make sure that this issue would be considered as a part of the Outreachy contribution. Thank you

**Comment by thibaudcolas** [comment]
@nouman17610 yes

**Comment by VictoriaAjala** [comment]
Hello @nouman17610 Can I confirm that you are still working on this issue...If not, Is it okay If I work on this?

**Comment by nouman17610** [comment]
@VictoriaAjala I am wokring on the issue. Thank you

**Comment by nouman17610** [comment]
@thibaudcolas I have reached this level. Please allow me editor access so I can complete this task and the final tasks of the application. Thank you

**Comment by nouman17610** [comment]
Draft for Procedure: How to disable a user account.

**How to disable a user account by making it active**
Making a user inactive allows you to keep the user's data and limit the user's access.

**Steps** 

1. Go to **Settings** on the left sidebar, then click **Users** in the sub-menu to access the Users page.
The **Users** page displays the list of users. The users list displays the active or inactive status on the right side of each user.

![The_users_listing_with_search_and_a_CTA_to_add width-900_4SFFPRC](https://github.com/wagtail/guide/assets/112282629/b251d439-1726-4188-840b-2e598067a4ca)

2. Put a cursor on the single user. A checkbox will appear on the left of the user name. Click the **Checkbox**.
A bulk action bar appears at the bottom of the page.

![bulk-action-bar](https://github.com/wagtail/guide/assets/112282629/cd4f6926-6ad8-4991-927c-217002111154)

3. Click the **Select active state** button.
Set active state appears with Active and Inactive radio buttons.

![Change-active-state-for-1-user-Wagtail](https://github.com/wagtail/guide/assets/112282629/fbd040ab-1153-4dcf-917d-e72463cefc3e)

4. Click the **Inactive** radio button, and then click **Yes, change state**.
On click action, it redirects to the **Users** page. When you scroll down to the user in the list, you see the status as active.  

![Change-active-state-for-1-user-Wagtail](https://github.com/wagtail/guide/assets/112282629/b3785b1c-6ba9-406b-a09f-1b0880ccd6ec)

The inactive user cannot access the account, but the user's data is safe.


@thibaudcolas 
Step 1 and step 2 are already available on https://guide.wagtail.org/en-latest/how-to-guides/manage-users-and-roles/.
We can start from step 3 if this procedure will be part of the **Manage users and roles** Guide
